docs/user/kubevirt/install-config.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: v1
+baseDomain: Enter here the base domain
+compute:
+- architecture: amd64
+  hyperthreading: Enabled
+  name: worker
+  platform: {}
+  replicas: 3
+controlPlane:
+  architecture: amd64
+  hyperthreading: Enabled
+  name: master
+  platform: {}
+  replicas: 3
+metadata:
+  creationTimestamp: null
+  name: Enter here the cluster name
+networking:
+  clusterNetwork:
+  - cidr: 10.128.0.0/14
+    hostPrefix: 23
+  machineNetwork:
+  - cidr: Enter here the machine network CIDR
+  networkType: OpenShiftSDN
+  serviceNetwork:
+  - 172.30.0.0/16
+platform:
+  kubevirt:
+    apiVIP: Enter here the API virtual IP
+    ingressVIP: Enter here the Ingress virtual IP
+    namespace: Enter here the namespace to install the tenant cluster resources in
+    networkName: Enter here the name of the network to be used by the tenant cluster
```

Comment:
Maybe be more detailed, something like "the network (NAD name) to be used ..." ?

Reply:
Changed